package.json

```diff
@@ -8,7 +8,8 @@
 		"test": "turbo run test --color",
 		"t": "pnpm test",
 		"test:types": "turbo run test:types --color",
-		"lint": "concurrently -n eslint,dprint \"eslint --ext ts .\" \"dprint check --list-different\""
+		"lint": "concurrently -n eslint,dprint \"eslint --ext ts .\" \"dprint check --list-different\"",
+		"lint:fix": "npx dprint fmt"
```

Comment:
This commands runs `dprint format` across all files, fixing the linting in the whole directory